shared/actions/tracker.js

```diff
@@ -93,26 +93,45 @@ export function registerTrackerIncomingRpcs (): TrackerActionCreator {
   }
 }
 
-export function triggerIdentify (uid: string): TrackerActionCreator {
+export function getProfile (username: string): TrackerActionCreator {
+  return (dispatch, getState) => {
+    dispatch(triggerIdentify('', username, true, serverCallMap(dispatch, getState, true), false, true, true))
+  }
+}
+
+export function getMyProfile (): TrackerActionCreator {
+  return (dispatch, getState) => {
+    const status = getState().config.status
+    const myUID = status && status.user && status.user.uid
+    if (myUID) {
+      dispatch(triggerIdentify(myUID, '', true, serverCallMap(dispatch, getState, true), false, true, true))
+    }
+  }
+}
+
+export function triggerIdentify (uid: string = '', userAssertion: string = '', allowSelf: boolean = false, incomingCallMap: Object = {},
```

Comment:
This is a lot of positional boolean props. Would it make sense to use the destructuring object style so that the names are explicit to the caller? E.g. `triggerIdentify({allowSelf: true, ...})`.